refresh/index.js

```diff
@@ -1008,6 +1009,12 @@ module.exports = generators.Base.extend({
             this.destinationPath('Sources', this.applicationModule, 'Extensions', 'WatsonConversationExtension.swift')
           );
         }
+        if(serviceType === 'alertnotification') {
+          this.fs.copy(
+            this.templatePath('extensions', 'AlertNotificationExtension.swift'),
+            this.destinationPath('Sources', this.applicationModule, 'Extensions', 'AlertNotificationExtension.swift')
+          );
+        }
```

Comment:
This block is not checking for `this._ifNotExistsInProject()` (the watson conversation block is also missing this check). We need to be careful not to overwrite files that already exists if they are in the Application module -- as this affects user controlled files in the CRUD flow). Can you fix the alert notification code here, but leave the watson conversation code as is -- we should open a separate github issue and PR to fix that one.